src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import {
+  calcEndItem,
+  calcPages,
+  calcStartItem,
+  getNumbers,
+} from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pages = calcPages(total, perPage);
```

Comment:
```suggestion
  const pagesСount = calcPages(total, perPage);
```
suggest more meaningful name